### DISCUSSION TITLE
iOS 16

### DISCUSSION THREAD

**Discussion by AntonShepotko** [community/community#33229] [Mobile]
Please add widgets for Lock Screen  iOS 16, and widget with PR list, I think it's will be cool. Thanks😎
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.80.0 (78250505)
Device: iPhone 13 Pro (iOS 16.0)
</details>

**Comment by eliperkins** [comment]
Hey @AntonShepotko! Thanks a ton for this feedback. Getting feedback like this from our customers definitely helps validate some of the ideas we've been noodling on related to new iOS 16 features.

I agree that having Lock Screen widgets and more would be super cool! Stay tuned. 😎
